### PR TITLE
separate supplemental resources from materials

### LINKED ISSUE #832: separate supplemental resources from required materials in 10 files (Part 4)
On one side, some apprentices think that they need to throughly look through every material, and are getting way behind because of it.  On the other, many well-meaning volunteers have thrown the kitchen sink into the "materials" section, when many or even all of them should be considered "supplemental resources" (supplemental meaning additional, extra, not required). 

Look through topic outlines, and change them all so that:
- the "slides" are part of the "materials" section
- the "lesson" explicitly directs how to use the mandatory "materials" (ex: "Read through [these slides]() )
- optional materials are under a "supplemental resources" section.
- if you finish, or notice a topic outline looks fine, please check it off on this list or let a Techtonica admin know it's complete.

Review these outlines:
- [ ] https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-9.md
- [ ] https://github.com/Techtonica/curriculum/blob/master/api/apis-and-json.md
- [ ] https://github.com/Techtonica/curriculum/blob/master/ajax/ajax.md
- [ ] https://github.com/Techtonica/curriculum/blob/master/api/twilio-api.md
- [ ] https://github.com/Techtonica/curriculum/blob/master/data-visualization/data-visualization.md
- [ ] https://github.com/Techtonica/curriculum/blob/master/express-js/express.md
- [x] https://github.com/Techtonica/curriculum/blob/master/databases/data-models.md
- [x] https://github.com/Techtonica/curriculum/blob/master/command-line/env.md
- [ ] https://github.com/Techtonica/curriculum/blob/master/databases/sql.md 
- [ ] https://github.com/Techtonica/curriculum/blob/master/databases/database-migrations.md

### PR DESCRIPTION
**Pull request by vsharma20v**
Separate supplemental resources from the material section of the following files:
- [x] https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-9.md
- [x] https://github.com/Techtonica/curriculum/blob/master/api/apis-and-json.md
- [x] https://github.com/Techtonica/curriculum/blob/master/ajax/ajax.md
- [x] https://github.com/Techtonica/curriculum/blob/master/api/twilio-api.md
- [x] https://github.com/Techtonica/curriculum/blob/master/data-visualization/data-visualization.md
- [x] https://github.com/Techtonica/curriculum/blob/master/express-js/express.md
- [x] https://github.com/Techtonica/curriculum/blob/master/databases/sql.md
- [x] https://github.com/Techtonica/curriculum/blob/master/databases/database-migrations.md

and I find the outline of the following file looks fine:
- [x] https://github.com/Techtonica/curriculum/blob/master/databases/data-models.md
- [x] https://github.com/Techtonica/curriculum/blob/master/command-line/env.md

Issue Fixed #832

### CODE REVIEW

File: ajax/ajax.md [hunk 1]
```diff
@@ -14,20 +14,22 @@ About 3 hours
...
 ### Objectives
 
 **Participants will be able to:**
 - form AJAX calls to an API.
 
-### Lesson
-[Slides](https://docs.google.com/presentation/d/1S3BjcLZNjex2_qiA9MdyJOjWZ_qmJ78STbUeDEyHH_8/edit#slide=id.g241461b869_0_5) | [Slides with Audio](https://drive.google.com/file/d/1pLyMqvC-8cFroVTWMUBeG1_BRd4L-rL3/view?usp=sharing)
-
 ### Materials
+- [AJAX Slides](https://docs.google.com/presentation/d/1S3BjcLZNjex2_qiA9MdyJOjWZ_qmJ78STbUeDEyHH_8/edit#slide=id.p)
```
Comment (alodahl) [inline ajax/ajax.md]: 🚩

File: ajax/ajax.md [hunk 2]
```diff
@@ -57,7 +59,7 @@ AJAX, however, enabled the browser to make requests and only re-render *parts* o
 
 When your JavaScript code runs in the browser, it does so in a *single thread of execution*. What this means is that it runs your code in sequence, one command at a time. It cannot, for example, take two different lines in your code and run them simultaneously.
 
-To understand why asynchrony is an important feature of AJAX, we can think about a synchronous scenario. Let's return to the example above of the end user visiting an e-commerce site. Imagine that the end user clicks on the "Buy" button to put an item in their shopping cart. This code might look something like this:
+To understand why asynchrony is an important feature of AJAX, we can think about an asynchronous scenario. Let's return to the example above of the end user visiting an e-commerce site. Imagine that the end user clicks on the "Buy" button to put an item in their shopping cart. This code might look something like this:
```
Comment (alodahl) [inline ajax/ajax.md]: 💙

File: api/twilio-api.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ Twilio is used at many companies to send text messages, and learning Twilio will
 ### Materials
 
 - [Twilio Quest](https://www.twilio.com/quest/welcome)
-- [Twilio Quest Tutorial Video (Scroll to Bottom of Page)](https://www.twilio.com/quest/welcome)
+- [Twilio Quest Tutorial Video (14min watch)](https://www.youtube.com/watch?v=ZzAEAfYw2xc)
```
Comment (alodahl) [inline api/twilio-api.md]: Can you also leave a note:
(video is at the bottom of this page)

File: data-visualization/data-visualization.md [hunk 4]
```diff
@@ -211,7 +211,7 @@ const g = d3.select("svg")
 - [Data Visualization – How to Pick the Right Chart Type?](https://eazybi.com/blog/data_visualization_and_chart_types/)
 - [Data Visualization 101: How to Choose the Right Chart or Graph for Your Data](https://blog.hubspot.com/marketing/types-of-graphs-for-data-visualization)
 
-#### For inspiration and general related reading
+#### For inspiration and generally related reading
```
Comment (alodahl) [inline data-visualization/data-visualization.md]: maybe just take out the word 'generally', it doesnt help the title much.

File: javascript/javascript-9.md [hunk 5]
```diff
@@ -27,23 +27,20 @@
...
 
-### Supplemental Materials
-- [Using Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)
-- [Understanding Promises](https://www.codecademy.com/courses/asynchronous-javascript/lessons/promises/exercises/understanding-promises)
-- [JavaScript Promises in 10 Minutes](https://www.youtube.com/watch?v=DHvZLI7Db8E)
-- [Understanding Promises in JavaScript](https://teamtreehouse.com/library/understanding-promises-in-javascript)
-- [Async/await reference](https://javascript.info/async-await)
-- [Introduction to Promises](https://beta.observablehq.com/@mbostock/introduction-to-promises)
+### Materials
+- [Video on Callbacks, Promises, Async-Await (25min watch)](https://youtu.be/PoRJizFvM7s)
```
Comment (Kundan28) [inline javascript/javascript-9.md]: You can provide the timestamp for all of the three..

File: api/twilio-api.md (hunk 3 above)
Comment (vsharma20v) [reply api/twilio-api.md]: But i directly mention the youtube link of the video so i think there is no need to mention this.
But if u want this i can do this

File: javascript/javascript-9.md (hunk 5 above)
Comment (vsharma20v) [reply javascript/javascript-9.md]: @Kundan28 done

File: data-visualization/data-visualization.md (hunk 4 above)
Comment (vsharma20v) [reply data-visualization/data-visualization.md]: @alodahl ma'am done

**Pull request merged**